src/frontend/Komponenter/Behandling/Fanemeny/sider.ts

```diff
@@ -113,6 +118,9 @@ export const filtrerSiderEtterBehandlingstype = (behandling: Behandling): ISide[
     if (behandling.behandlingsårsak === Behandlingsårsak.G_OMREGNING) {
         return sider.filter((side) => filtrerHvisGOmregning.includes(side.navn as SideNavn));
     }
+    if (behandling.behandlingsårsak === Behandlingsårsak.SATSENDRING) {
```

Comment:
Kan vi gjenbruke de for `filtrerHvisGOmregning` i stedet? Så lager vi ikke duplikat for det samme

Reply:
Kan gjenbruke og rename til `filtrerHvisGOmregningEllerSatsendring`